AEPLifecycle/Tests/FunctionalTests/LifecycleFunctionalTests.swift

```diff
@@ -38,6 +38,7 @@ class LifecycleFunctionalTests: XCTestCase {
         mockRuntime.ignoreEvent(type: EventType.lifecycle, source: EventSource.applicationClose)
         mockRuntime.ignoreEvent(type: EventType.lifecycle, source: EventSource.applicationLaunch)
         UserDefaults.clear()
```

Comment:
nit: 
Can you replace the following two lines in our tests 
```
UserDefaults.clear()
NamedCollectionDataStore.clearStorageFiles()
```
to single API call. 
```
NamedCollectionDataStore.clear()
```

Reply:
fixed